### ISSUE TITLE
Field description

### ISSUE THREAD

**Issue by venelin-mihaylov** [hasura/graphql-engine#1434]
Is it possible to use the column description to populate the description field in the graphql schema? 
If not, can you please implement that feature?

**Comment by shahidhk** [comment]
Hello @venelin-mihaylov, this is a duplicate of the issue https://github.com/hasura/graphql-engine/issues/446

We'll start working on the issue once certain code refactoring is complete, which could take a couple of weeks.

Thanks for creating the issue, but I will be closing this now in favour of the existing one.

**Comment by venelin-mihaylov** [comment]
Thank you
